### PR TITLE
Redoing the "Classification" and "Download" Pages

### PR DESCRIPTION
**Pull request by DrKJeff16**

### CODE REVIEW

**Comment by DrKJeff16** [comment]
I'm still missing the Download Page.

File: dist/classification.html [hunk 1]
```diff
@@ -44,118 +45,159 @@
...
+	<main><!--Main Body.-->
+		<article> <!--TODO Should we add a background? (style="background: url('img/tosdr-logo-128-low-opacity.png') no-repeat center;")-->
+			<section id="process">
+				<div class="container" id="tosdr-content" role="main">
+					<h2><strong>How we review</strong></h2>
+					<p>
+						When a new service is added to the database, its Terms are reviewed by <ins>contributors</ins>, who divide the pertinent documents (<strong>ToS</strong>, <strong>Privacy Policy</strong>, <strong>Cookie Policies</strong>, etc.) into small quotes ("<ins>points</ins>"), each referring to a specific <a href="https://edit.tosdr.org/cases" target="_blank"><strong><ins>Case</ins></strong></a> which will assign one of the following badges to the point:
+					</p>
+					<div>
+						<span class="badge badge-success"><i class="icon-thumbs-up icon-white">+</i> Good</span>		<br>
```
Comment (JustinBack) [inline dist/classification.html:87]: There are missing whitespaces at the end of the text

**Review by JustinBack — CHANGES_REQUESTED** [review]
Found something nothing to worry about, but it would be great to fix design wise

File: dist/classification.html (hunk 1 above)
Comment (DrKJeff16) [reply dist/classification.html]: All good now :3

**Comment by DrKJeff16** [comment]
Hmmm what's missing there? :thinking:

File: src/classification.html [hunk 2]
```diff
@@ -44,118 +45,191 @@
...
+					</p>
+					<div id="ratings">
+						<span class="badge badge-success"><i class="icon-thumbs-up icon-white">+</i> Good&nbsp;</span>
+						<br>
+						<span class="badge badge-warning"><i class="icon-thumbs-down icon-white">-</i> Bad&nbsp;</span>
+						<br>
+						<span class="badge badge-important"><i class="icon-remove icon-white">×</i> Blocker&nbsp;</span>
+						<br>
+						<span class="badge"><i class="icon-arrow-right icon-white">→</i> Neutral&nbsp;</span>
+					</div>
```
Comment (JustinBack) [inline src/classification.html:94]: Still missing whitespaces, non breaking spaces wont work here.
```suggestion
					<div id="ratings">
						<span class="badge badge-success"><i class="icon-thumbs-up icon-white">+</i> Good </span>
						<br>
						<span class="badge badge-warning"><i class="icon-thumbs-down icon-white">-</i> Bad </span>
						<br>
						<span class="badge badge-important"><i class="icon-remove icon-white">×</i> Blocker </span>
						<br>
						<span class="badge"><i class="icon-arrow-right icon-white">→</i> Neutral </span>
					</div>
```

**Review by JustinBack — CHANGES_REQUESTED** [review]
Found 1 issue and you have conflicts in the API. You need to pull from master again.

**Review by JustinBack — APPROVED** [review]

File: dist/api/1/all.json [hunk 3]
```diff
@@ -1,9 +1,71 @@
...
   "tosdr/api/version": 1,
+<<<<<<< HEAD
+<<<<<<< HEAD
+<<<<<<< HEAD
+<<<<<<< HEAD
+<<<<<<< HEAD
+<<<<<<< HEAD
+<<<<<<< HEAD
+<<<<<<< HEAD
+<<<<<<< HEAD
```
Comment (JustinBack) [inline dist/api/1/all.json:11]: Please remove the merge comments :-)

File: package-lock.json [hunk 4]
```diff
@@ -3011,9 +3011,15 @@
...
     "ini": {
+<<<<<<< HEAD
       "version": "1.3.7",
       "resolved": "https://registry.npmjs.org/ini/-/ini-1.3.7.tgz",
       "integrity": "sha512-iKpRpXP+CrP2jyrxvg1kMUpXDyRUFDWurxbnVT1vQPx+Wz9uCYsMIqYuSBLV+PAaZG/d7kRLKRFc9oDMsH+mFQ==",
+=======
+      "version": "1.3.8",
+      "resolved": "https://registry.npmjs.org/ini/-/ini-1.3.8.tgz",
+      "integrity": "sha512-JV/yugV2uzW5iMRSiZAyDtQd+nxtUnjeLt0acNdw98kKLrvuRVyB80tsREOE7yvGVgalhZ6RNXCmEHkUKBKxew==",
+>>>>>>> Compiled.
```
Comment (JustinBack) [inline package-lock.json:3022]: Merge comments break json

File: src/classification.html (hunk 2 above)
Comment (JustinBack) [inline src/classification.html:94]: Non breaking spaces are back again :-)

**Review by JustinBack — CHANGES_REQUESTED** [review]

**Pull request merged**